src/nvim/lua/stdlib.c

```diff
@@ -541,6 +542,32 @@ static int nlua_iconv(lua_State *lstate)
 
 #endif
 
+int nlua_read_secure(lua_State *L)
+{
+  if (lua_gettop(L) < 1) {
+    return luaL_error(L, "Expected 1 argument");
+  }
+
+  if (lua_type(L, 1) != LUA_TSTRING) {
+    luaL_argerror(L, 1, "expected string");
+  }
+
+  const char *contents = NULL;
+  size_t len = 0;
+  const char *path = lua_tolstring(L, 1, &len);
+  if (path != NULL) {
+    contents = read_secure(path, &len);
+  }
+
+  if (contents != NULL) {
+    lua_pushlstring(L, contents, len);
+    xfree((void *)contents);
```

Comment:
When `read_secure` no longer returns `const`, `contents` can just be `char*` and the cast here will no longer be necessary.
```suggestion
    xfree(contents);
```

Reply:
Fixed in 4365a6f3f4319764aac66ff814848253e50b6fee